### PR TITLE
Add printing of lifecycle ExamplesTable to JSON output

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java [hunk 1]
```diff
@@ -419,6 +423,7 @@ public void beforeExamples(List<String> steps, ExamplesTable table) {
         }
         print(format("examplesStepsEnd", EMPTY));
         print(formatTable(table));
+        print(format("examplesStart", EMPTY));
```
Comment (valfirst) [inline jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java:426]: looks strange

Comment (valfirst) [inline jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java]: if we have `examplesStart` then we should also have `examplesEnd`, shouldn't we?

Comment (uarlouski) [reply jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java]: renamed to examplesHeader to not confuse

File: jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java [hunk 2]
```diff
@@ -419,6 +423,7 @@ public void beforeExamples(List<String> steps, ExamplesTable table) {
         }
         print(format("examplesStepsEnd", EMPTY));
         print(formatTable(table));
+        print(format("exampleScenarioStart", EMPTY));
```
Comment (valfirst) [inline jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java]: `exampleScenariosStart`

File: jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java [hunk 3]
```diff
@@ -431,6 +436,7 @@ public void example(Map<String, String> tableRow, int exampleIndex) {
 
     @Override
     public void afterExamples() {
+        print(format("exampleScenarioEnd", EMPTY));
```
Comment (valfirst) [inline jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java]: `exampleScenariosEnd`

**Review by valfirst — CHANGES_REQUESTED** [review]

File: jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java (hunk 2 above)
Comment (valfirst) [reply jbehave-core/src/main/java/org/jbehave/core/reporters/PrintStreamOutput.java]: also I propose to introduce this change in master repo as well

**Review by valfirst — APPROVED** [review]

**Pull request merged**